### PR TITLE
CON-2557-Markdown-Reverse-Array-exercise

### PR DESCRIPTION
**Pull request by zanninso**

### CODE REVIEW

File: subjects/java/checkpoints/reverse-array/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## ReverseArray
+
+### Instructions
+
+In a file named `ReverseArray.java` write a function `reverse` that reverses the given array specified in the parameters and returns it.
```
Comment (nprimo) [inline subjects/java/checkpoints/reverse-array/README.md:5]: I would rephrase this sentence to improve its clarity

```
Create a Java file named `ReverseArray.java` and define a function named `reverse` inside it. This function should take an array as input, reverse the order of its elements, and return the new reversed array.
```

**Review by nprimo — CHANGES_REQUESTED** [review]

File: subjects/java/checkpoints/reverse-array/README.md (hunk 1 above)
Comment (zanninso) [reply subjects/java/checkpoints/reverse-array/README.md]: Nice suggestion, Thank you

Comment (zanninso) [reply subjects/java/checkpoints/reverse-array/README.md]: done

**Review by nprimo — APPROVED** [review]
LGTM
